activerecord/test/cases/associations/required_test.rb

```diff
@@ -22,30 +22,6 @@ class Child < ActiveRecord::Base
     @connection.drop_table "children", if_exists: true
   end
 
-  test "belongs_to associations are not required by default" do
```

Comment:
I got it... I didn't notice that the we should set  `Rails.application.config.active_record.belongs_to_required_by_default = true` to make `belongs_to` required by default.

It's the default setting on fresh installed Rails5, but `nil` when upgrade to Rails5... so I don't know if that PR is relevant 😢 

But at the same way, I think that those tests doesn't do exacly what they tell they test... I mean, they really test if a `belongs_to` association is required or not by default... the problem is that "default" need to be set and the test doesn't consider it...